doc/api/buffer.md

```diff
@@ -2308,6 +2308,7 @@ added: v0.1.90
 * {integer}
 
 Returns the number of bytes in `buf`.
+An alias of the inherited [`buf.byteLength`][].
```

Comment:
I think it's more correct to say it's the inherited [`buf.length`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/length) – which happens to be the same as `buf.byteLength`. I would remove this change from this PR.